python/dgl/optim/pytorch/sparse_optim.py

```diff
@@ -130,16 +128,21 @@ def step(self):
                                     < idx_i.shape[0]:
 
                                 if idx_shmem_name in self._shared_cache[emb_name]:
-                                    shmem_ptr_holder.append(
+                                    self.shmem_buffer_holder.append(
                                         self._shared_cache[emb_name][idx_shmem_name])
-                                    shmem_ptr_holder.append(
+                                    self.shmem_buffer_holder.append(
                                         self._shared_cache[emb_name][grad_shmem_name])
 
-                                # in case idx_i.shape[0] is 0
+                                # The minimun buffer size is 32.
```

Comment:
why is the minimum buffer size so small? I think the buffer size can start from a couple of MB.